server/apps/research/views.py

```
@@ -22,6 +22,7 @@
 from rest_framework.throttling import UserRateThrottle
 from django.core.cache import cache
 from rest_framework.pagination import PageNumberPagination
+from apps.research.models import Category
```

Ruff (0.8.2)

Redefinition of unused `Category` from line 12

(F811)